socorro/unittest/external/boto/test_connection_context.py

```diff
@@ -0,0 +1,206 @@
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import mock
+import json
+import tempfile
+import shutil
+from os.path import join
+
+import boto.exception
+
```

Comment:
If your intention is to "linebreak bucketize" the imports by built-ins and third-party and socorro etc. then the `import mock` should be together with `import boto.exception`